src/Umbraco.Core/Models/IGridAttributes.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using System.Collections.Generic;
+
+namespace Umbraco.Core.Models
+{
+    public interface IGridAttributes
+    {
+        /// <summary>
+        /// Gets or sets the configuration.
+        /// </summary>
+        /// <value>
+        /// The configuration.
+        /// </value>
+        [JsonProperty("config")]
+        JToken Config { get; set; }
+
+        /// <summary>
+        /// Gets or sets the styles.
+        /// </summary>
+        /// <value>
+        /// The styles.
+        /// </value>
+        [JsonProperty("styles")]
+        JToken Styles { get; set; }
+
+        /// <summary>
+        /// Gets the attributes.
+        /// </summary>
+        /// <returns></returns>
+        Dictionary<string, string> GetAttributes();
```

Comment:
Should be `IDictionary<string, string>`

Reply:
@dbardi did you create an issue on the issue tracker http://issues.umbraco.org that @Shazwazza mentioned?

If so can you please update this PR with the link or issue number.
Thanks,
Warren :)

Reply:
This has been done: http://issues.umbraco.org/issue/U4-10520
I've also closed this PR and created a new one here: https://github.com/umbraco/Umbraco-CMS/pull/2234